node_modules/oae-core/topnavigation/topnavigation.html

```diff
@@ -41,10 +21,26 @@ <h1 id="topnavigation-target" class="sr-only">__MSG__SKIP_MENU_TARGET__</h1>
 //////////////////////////////
 -->
 
+<div id="topnavigation-anonymous-template"><!--
```

Comment:
Would it make sense to move the anonymous template into a single template rather than having subtemplates? The same is true for the logged in template.